tests/integration/ha_tests/helpers.py

```diff
@@ -734,6 +735,58 @@ async def update_pebble_plans(ops_test: OpsTest, override: Dict[str, str]) -> No
         assert ret_code == 0, f"Failed to replan for unit {unit.name}"
 
 
+async def reused_storage(ops_test: OpsTest, reused_unit: Unit, removal_time: datetime) -> None:
+    """Cats the k8s logs, looking for messages related to startup state reusing the storage.
+
+    MongoDB startup message indicates storage reuse:
+        If member transitions to STARTUP2 from STARTUP then it is syncing/getting data from
+        primary.
+        If member transitions to STARTUP2 from REMOVED then it is re-using the storage we
+        provided.
+    """
+    cat_cmd = [
+        "ssh",
+        "--container",
+        MONGODB_CONTAINER_NAME,
+        reused_unit.name,
+        "cat /var/lib/mongodb/mongodb.log",
+    ]
+
+    return_code, start_states, _ = await ops_test.juju(*cat_cmd)
+
+    assert (
+        return_code == 0
+    ), f"Failed catting mongodb logs, unit={reused_unit.name}, container={MONGODB_CONTAINER_NAME}"
+
+    for line in start_states.split("\n"):
+        if not len(line):
+            continue
+
+        item = json.loads(line)
+
+        re_use_time = convert_time(item["t"]["$date"])
```

Comment:
could you please change variable name to "reuse_time" because the first association with "re_*" is regex